src/layouts/InterfaceLayout/components/InterfaceMobileMenuComponent/InterfaceMobileMenuComponent.scss

```diff
@@ -0,0 +1,47 @@
+@import "~@/scss/GlobalVariables";
+
+* {
+  font-family: sans-serif;
+  user-select: none;
+}
+
+.interface-mobile-menu {
+  background-color: white;
+  padding-bottom: 50px;
+}
+
+.menu-group {
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 20px;
+  margin-top: 20px;
+  border-top: 1px solid rgba(0, 0, 0, 0.03);
+}
+
+.active {
+  color: black !important;
+  > p {
+    color: black !important;
+  }
+}
+
+.top-menu {
+  > p {
+    img {
+      margin-right: 10px;
+    }
+
+    color: #c1c1c1;
+    font-size: 25px;
+    font-weight: 600;
+  }
+}
+
+.sub-menu {
+  > div {
```

Comment:
you may want to add cursor: pointer here so that the user knows that its clickable!

Reply:
Fixed.